tap_sftp/client.py

```diff
@@ -75,6 +91,10 @@ def __exit__(self):
 
     def close(self):
         if self.__active_connection:
+            # get socket
```

Comment:
Please elaborate more, why do we need to get a socket?

Reply:
To set the timeout we need socket.